### DISCUSSION TITLE
Try to flash Lilygo T Beam Supreme

### DISCUSSION THREAD

**Discussion by AlexanderGabriel** [adafruit/tinyuf2#397] [Q&A] [answered]
Hi,

i try to get tiyuf2 running on a Lilygo T Beam Supreme.
I use esptool to flash downloaded release "tinyuf2-lilygo_ttgo_tbeam_s3-0.18.2.zip":
esptool.py --chip esp32s3 -p /dev/ttyACM0  -b 460800 --before=default_reset --after=hard_reset write_flash --flash_mode dio --flash_freq 80m --flash_size 8MB 0x8000 partition-table.bin 0xe000 ota_data_initial.bin 0x1000 bootloader.bin 0x2d0000 tinyuf2.bin

after this, i reset and get errors on serial console
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0x7 (TG0WDT_SYS_RST),boot:0x8 (SPI_FAST_FLASH_BOOT)
Saved PC:0x40043ac8
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x3ac
load:0x403c9700,len:0x9b4
load:0x403cc700,len:0x28d8
Checksum failure. Calculated 0x5b stored 0x60
ets_main.c 329 
ESP-ROM:esp32s3-20210327
Build:Mar 27 2021
rst:0x7 (TG0WDT_SYS_RST),boot:0x8 (SPI_FAST_FLASH_BOOT)
Saved PC:0x40043ac8
SPIWP:0xee
mode:DIO, clock div:1
load:0x3fce3808,len:0x3ac
load:0x403c9700,len:0x9b4
load:0x403cc700,len:0x28d8
Checksum failure. Calculated 0x5b stored 0x60
ets_main.c 329 

How can i get tinyuf2 running?

Thank you.

**Comment by VirusPilot** [comment, marked as answer]
I would rather download the following: https://github.com/adafruit/tinyuf2/actions/runs/9785406115/artifacts/1665865293 and from this package just flash "combined.bin" at 0x0 using e.g. https://espressif.github.io/esp-launchpad/ in Chrome. After a reset your S3 will boot in UF2 mode and wait for a UF2 image to be loaded (e.g. SoftRF).

Best from Würzburg@Germany!

**Reply by AlexanderGabriel** [reply]
Thank you. SoftRF was my goal and your answer is exactly what i needed :)

Best from Bruchsal@Germany :)

**Reply by VirusPilot** [reply]
Cool. You may then actually consider my SoftRF fork where you can modify the ICAO hex code: https://github.com/VirusPilot/SoftRF. I am providing UF2 versions as described here: https://github.com/VirusPilot/SoftRF/tree/master?tab=readme-ov-file#uf2-binaries
